conveyal/analysis-ui
# Fix remove stops pattern selector

## How to test

1. Create a Remove Stops modification.
2. Add and remove Patterns to ensure it works as intended.
